src/app/lib/streamer.js

```diff
@@ -509,9 +518,9 @@
         },
     };
 
-    var streamer = new WebTorrentStreamer();
+    App.Streamer = new WebTorrentStreamer();
```

Comment:
seems I forgot about that. It's just to have access to the streamer from console, doesn't change anything